lua/nvim-treesitter/health.lua

```diff
@@ -7,13 +7,14 @@ local info = require'nvim-treesitter.info'
 local health_start = vim.fn["health#report_start"]
 local health_ok = vim.fn['health#report_ok']
 local health_error = vim.fn['health#report_error']
+local health_warn = vim.fn['health#report_warn']
 
 local M = {}
 
 local function install_health()
   health_start('Installation')
   if fn.executable('tree-sitter') == 0 then
-    health_error('`tree-sitter` executable not found')
+    health_warn('`tree-sitter` executable not found')
```

Comment:
but there is still the explanation missing what `tree-sitter` is used for

Reply:
space had add. the explanation of  tree-sitter like what?

Reply:
I'd say he meant `the command-line is required when running `TSInstallFromGrammar` and for the installation of some parsers`